doc/man3/SSL_CTX_set_msg_callback.pod

```diff
@@ -172,6 +173,9 @@ The pseudo content types B<SSL3_RT_QUIC_DATAGRAM>, B<SSL3_RT_QUIC_PACKET>,
 B<SSL3_RT_QUIC_FRAME_FULL>, B<SSL3_RT_QUIC_FRAME_HEADER> and
 B<SSL3_RT_QUIC_FRAME_PADDING> were added in OpenSSL 3.2.
 
+In versions previous to OpenSSL 3.3 I<cb> was called with 0 as I<version> for
```

Comment:
Hmm.... this could still make it to 3.2.

I would not change this in stable branches.

Reply:
It's also not quite accurate, since for TLS it was changed in 3.0.

Reply:
You are right. I will correct this.